conjur-api/Variable.cs

```diff
@@ -38,5 +39,25 @@ public string GetValue()
         {
             return this.Client.AuthenticatedRequest(this.path + "/value").Read();
         }
+
+        /// <summary>
+        /// Adds a value to the variable.
+        /// </summary>
+        public void AddValue (string value)
+        {
+            var req = this.Client.AuthenticatedRequest ($"{this.path}/values");
```

Comment:
Please keep the formatting consistent. In particular please don't use a space before the method parameters.

Reply:
Oh good catch! That's probably the default format in Mac's VS.